src/calendar/CalendarMonth.jsx

```diff
@@ -30,11 +24,39 @@ const CalendarMonth = React.createClass({
     onMonthChange: React.PropTypes.func,
     onYearChange: React.PropTypes.func,
     value: CustomPropTypes.momentOrMomentRange,
+    locale: React.PropTypes.string,
+  },
+
+  _setupLocale(locale) {
+    this.moment = moment();
+
+    if (locale !== 'en') {
+      require(`moment/locale/${locale}`);
```

Comment:
I think the user of this component will need to be responsible for requiring or registering their own `locale` definition with `moment`.